lib/plugins/create/templates/aws-ruby/.gitignore

```diff
@@ -0,0 +1,50 @@
+*.gem
```

Comment:
Actually.. counter to what @horike37 mentioned on my PR. I think this should be `gitignore` not `.gitignore`. Look at all the other runtime templets 😉

Reply:
https://github.com/serverless/serverless/blob/b04473bac1410c01aa1a8c1bdf2b82040f91d6ca/lib/plugins/create/create.js#L220-L222
I agree with @dschep on this one.
Other then that, great work @BigChief45, looking forward to use it.